templates/secrets/metadata-connection-secret.yaml

```diff
@@ -2,7 +2,12 @@
 ## Airflow Metadata Secret
 #################################
 {{- if (and .Values.data.metadataConnection (not .Values.data.metadataSecretName)) }}
-{{- $host := .Values.data.metadataConnection.host | default (printf "%s-%s.%s.svc.cluster.local" .Release.Name "postgresql" .Release.Namespace) }}
+{{- $postgres_host := .Values.data.metadataConnection.host | default (printf "%s-%s.%s.svc.cluster.local" .Release.Name "postgresql" .Release.Namespace) }}
+{{- $pgbouncer_host := (printf "%s-%s.%s.svc.cluster.local" .Release.Name "pgbouncer" .Release.Namespace) }}
+{{- $host := ternary $pgbouncer_host $postgres_host .Values.pgbouncer.enabled }}
+{{- $port := ((ternary .Values.ports.pgbouncer .Values.data.metadataConnection.port .Values.pgbouncer.enabled) | toString) }}
+{{- $database := (ternary (printf "%s-%s" .Release.Name "metadata") .Values.data.metadataConnection.db .Values.pgbouncer.enabled) }}
```

Comment:
Does this need to change for pgbouncer support? I thought the DB name is the same regardless?

Reply:
Guess this isn't a change thought

Reply:
@ashb that part doesn't NEED to change but frankly, the gnarliness of that line is what allowed this bug to exist in the first place. Figured it was good to do some housekeeping while I'm changing the file anyways.